src/reducer.spec.ts

```diff
@@ -182,6 +182,61 @@ describe(onNgrxForms.name, () => {
   });
 });
 
+describe(onNgrxForm.name, () => {
+  it('should call the reducer for controls', () => {
+    const state = {
+      prop: 'value',
+      form: INITIAL_STATE.controls.inner,
+    };
+
+    const resultState = onNgrxForm().reducer(state, new MarkAsTouchedAction(FORM_CONTROL_INNER_ID));
+    expect(resultState.form).not.toBe(INITIAL_STATE.controls.inner);
```

Comment:
while this is exactly the way I had written my tests, in this case this is wrong. if you see my other comment at your implementation of `onNgrxForm` you will realize that in this test case `resultState` will be `false` and `resultState.form` will be `undefined`, which makes the assertion succeed, but in the wrong way